submissions/models.py

```diff
@@ -274,3 +274,16 @@ def update_score_summary(sender, **kwargs):
                 u"Error while updating score summary for student item {}"
                 .format(score.student_item)
             )
+
+
+class ScoreAnnotation(models.Model):
+    """ Annotate individual scores with extra information if necessary. """
+
+    score = models.ForeignKey(Score)
+    # A string that will represent the 'type' of annotation,
+    # e.g. staff_override, etc.
+    annotation_type = models.CharField(max_length=255, blank=False, db_index=True)
+
+    # The anonymized User ID that the XBlock sees, not their real ID.
+    creator = models.CharField(max_length=255, blank=False, db_index=True)
```

Comment:
It might be good to introduce an AnonymizedUserIDField so that we can share validation. I'm presuming we also store anonymized ids in the submissions themselves.